index.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mfadhili Gitau</title>
+
+    <!-- Google Fonts  -->
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+
+    <!-- CSS file  -->
+    
+    <link rel="stylesheet" href="./css/main.css">
+</head>
+<body>
+    <header>
+       <div class="welcome">
+            <a class="welcome__text">Welcome</a>
```

Comment:
- [ ] You have an `a` tag without the attribute `href` that holds the path to visit upon click. Kindly fix this